### ISSUE TITLE
Geofire not compatible with 7.xx Firebase

### ISSUE THREAD

**Issue by charlie884** [shripalsoni04/nativescript-geofire-plugin#9]
I'm using the nativescript-plugin-firebase: ^7.7.0 

[!] CocoaPods could not find compatible versions for pod "Firebase/Database":
  In Podfile:
    Firebase/Database

    GeoFire (from `https://github.com/firebase/geofire-objc.git`, commit `96d6463f59016785f9c5ae9b669019c44786a23f`) was resolved to 2.0.0, which depends on
      Firebase/Database (~> 4.0)

Specs satisfying the `Firebase/Database, Firebase/Database (~> 4.0)` dependency were found, but they required a higher minimum deployment target.

**Comment by khlilelkhatatt** [comment]
Any answer ?

**Comment by charlie884** [comment]
@khlilelkhatatt No, but iwas looking and the plugin geofire apparently long time ago dont update, only works in nativescript 2 but i dont know why worked in android with nativescript 5, im looking other alternative if you find other way let my know please.
